### PR TITLE
Proofreading of hiring guide

### PR DESCRIPTION
**Pull request by ElenaFdR**

### CODE REVIEW

File: activities/recruitment/hiring-process.md [hunk 1]
```diff
@@ -1,36 +1,37 @@
...
+
+4\. Questions about the content of the position: route to CEO for answering
+5\. Questions about the hiring procedure and the conditions: route to HRM for answering
+6\. Applications are sent to HRM, for processing
+
+## Planned recruitment and hiring process
+
+7\. Current job openings have no closing date and applications are considered on a rolling basis. When we find a suitable candidate, we hire and close the opening. This means applications are considered ‘by themselves’ and not ‘in competition’ with a certain set of other applications.
+8\. HRM confirms receipt to candidates and does first round of evaluation.
+9\. HRM brings possibly suitable candidates to the attention of CEO and supervisor. These 2 make +/- judgement, without consulting each other. If judgement is not equal, a short discussion follows and judgement is made.
```
Comment (clausmullie) [inline activities/recruitment/hiring-process.md]: @MirjamvT what is meant by 'supervisor' here?

**Comment by ElenaFdR** [comment]
@bvhme My past experience is that breaking up a numbered list by adding a heading leads to Markdown restarting the numbering. If that's not necessary here, then I'll rework this.

File: activities/recruitment/hiring-process.md (hunk 1 above)
Comment (MirjamvT) [reply activities/recruitment/hiring-process.md]: the person who is hierarchically above the person ('leidinggevende')

**Comment by bvhme** [comment]
It is not necessary, also I believe we should limit us to the content formats presented to us by MarkDown and HTML in order to simplify our work and increase the reusability of our content.

So if this would not be a part of MarkDown I would suggest making a list with sub-lists instead of the current solution of making a paragraph which looks like a list but can't actually be styled/ingested as one. Is some more work now, and some things will not be as _nice_, but it will greatly improve how we can build on and improve this work.

**Pull request merged**